src/multi.jl

```diff
@@ -0,0 +1,77 @@
+# Univariate
+
+const FillVectorOfUnivariate{
+    S <: ValueSupport,
+    T <: UnivariateDistribution{S},
+    Tdists <: Fill{T, 1},
+} = VectorOfUnivariate{S, T, Tdists}
+
+function FillDist(dist::UnivariateDistribution, N::Int)
```

Comment:
Again I think this should rather be a lowercase function.

Reply:
Sounds good.